enigma-swing/src/main/java/cuchaz/enigma/gui/panels/right/AbstractInheritanceTree.java

```diff
@@ -46,12 +45,10 @@ private void onClick(MouseEvent event) {
 			}
 
 			Object node = path.getLastPathComponent();
-			if (node instanceof AbstractClassTreeNode classNode) {
-				gui.getController().navigateTo(new ClassEntry(classNode.getClassName()));
-			} else if (node instanceof AbstractMethodTreeNode methodNode) {
-				if (!(methodNode instanceof MethodInheritanceTreeNode inheritanceNode) || inheritanceNode.isImplemented()) {
-					gui.getController().navigateTo(methodNode.getMethodEntry());
-				}
+			if (node instanceof ClassInheritanceTreeNode classNode) {
+				gui.getController().navigateTo(classNode.getClassEntry());
+			} else if (node instanceof MethodInheritanceTreeNode methodNode && methodNode.isImplemented()) {
+				gui.getController().navigateTo(methodNode.getMethodEntry());
```

Comment:
Looks like there was an improper merge here, the class one should be `instanceof AbstractClassNode` and the method one `instanceof AbstractMethodTreeNode`, which should be always navigable to as opposed to an `InheritanceNode`, which should only navigable if it's implemented